### DISCUSSION TITLE
Fermata Auto not working anymore after android auto update.

### DISCUSSION THREAD

**Discussion by prw19** [AndreyPavlenko/Fermata#338] [Q&A]
Hi, I have a frustrated issue what I cant fix.
What happens:
Icon on the AA is gone. Nowhere find the app. It was working until 15 december. Now I cant use it anymore.

What I have try to resolve:.
1: Clean install 2 older android auto version
2: Clean install Fermata Auto And also Auto controll from 2 versions early
3: Reset all settings

Nothing will resolve the problem. 

Device info:
Samsung S23+ Android 14 UI.6.0
Volkswagen Discovery Pro 4G 2023

Please help me!!

**Comment by AndreyPavlenko** [comment]
See this issue - https://github.com/AndreyPavlenko/Fermata/issues/327

**Reply by prw19** [reply]
Hi, Im Sorry but I cant find the solution for me 🤔😔 and it still not working. The link what you have give me is a person that change something in the files but I do not have any experience to do that.
